packages/hardhat-plugin/src/index.ts

```diff
@@ -672,7 +676,7 @@ async function resolveConfigPath(
   try {
     const rawFile = await readFile(filepath);
 
-    return JSON.parse(rawFile.toString(), bigintReviver);
+    return await json5parse(rawFile.toString(), bigintReviver);
```

Comment:
small swap to camelCase to match the updated import above

```suggestion
    return await json5Parse(rawFile.toString(), bigintReviver);
```

Reply:
thanks for the review & precise instructions!

im not sure about the import part, the ignition scope at line 119 does not reach the function `resolveConfigPath` at line 672 and it gives an error due to scope.

i can add it around line 676, but im not entirely familiar with dynamic imports, not sure if it causes any trouble 🤔 

also, the function `resolveParametersString` seems to parse JSON as well, should this use JSON5 too?

Reply:
Ah! My mistake. That's what I get for reviewing a PR at 3am 😅

Don't worry about the import, it's fine where it is. Could you change the JSON parsing in `resolveParametersString` as well, then this should be ready to merge assuming the CI passes. Thanks!